src/controllers/expenses.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+
+const getAllExpenses = (req, res) => {
+  const expenses = expensesService.getAllExpenses();
+
+  res.send(expenses);
+};
+
+const getExpenseById = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expensesService.getExpenseById(Number(expenseId));
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const addNewExpense = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  if (typeof userId !== 'number'
+    || typeof spentAt !== 'string'
+    || typeof title !== 'string'
+    || typeof amount !== 'number'
+    || typeof category !== 'string'
+    || typeof note !== 'string'
+  ) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expensesService.addNewExpense(
+    Number(userId),
+    spentAt,
+    title,
+    Number(amount),
+    category,
+    note,
+  );
```

Comment:
What's the point of using typecasting here if you have a checking of types above?